portfolio/styles.css

```diff
@@ -0,0 +1,73 @@
+
+*{
+    margin: 0;
+    padding:0;
```

Comment:
I would remove the `margin:0` and `padding:0` rules I don't think they are affecting your styles